check-diff.sh

```diff
@@ -395,7 +404,11 @@ function install_tools {
 			npm install -g jscs
 		fi
 
-		# @todo ESLint
+		# Install ESLint
+		if [ -n "$ESLINT_CONFIG" ] && [ -e "$ESLINT_CONFIG" ] && [ "$( type -t eslint )" == '' ] && ! grep -sqi 'eslint' <<< "$DEV_LIB_SKIP"; then
```

Comment:
@westonruter You check for errors with the config file using the `-e` flag here but not below when running ESLint, any particular reason or should it be added below?